### DISCUSSION TITLE
Will meteorological forecast be available for the competition?

### DISCUSSION THREAD

**Discussion by ManuelAngel99** [PaddlePaddle/PaddleSpatial#9] [BaiduKDDCup2022] [answered]
Dear organizers,

I have read that no external data sources will be allowed without the explicit permission of the organizers. For that reason, I was wondering whether meteorological forecasts will be made available for the predictions / allowed as a data source.

Thank you for your time.

Manuel.

**Comment by zhoujingbo** [comment, marked as answer]
Hi ManuelAngel, 

The meteorological forecast data will not be provided and not be allowed as a data source. 

Traditionally, the meteorological forecast data is often adopted for long-term wind power forecasting. In our settings, we aim to forecast the short-term (within 1~2 days) power generated by a wind farm without knowing the future weather conditions, which is more challenging but more applicable for the majority. In particular, the 15-min meteorological forecast data is often unavailable. Therefore, no meteorological forecast data will be provided in the future full dataset. 

Moreover, the exact location and time period of the collected data will not be provided due to safety concerns. 

Thanks for your understanding.

**Comment by zhrli** [comment]
but if we knew the location and time，meteorological data could be available on time from public sources.

**Reply by zhoujingbo** [reply]
Hi,
The exact location and time will not be disclosed. Thanks. 

Regards
Jingbo
